crates/pathfinder/src/hub.rs

```diff
@@ -279,35 +254,35 @@ mod tests {
         assert_eq!(stream.data, Bytes::from(vec![0x01, 0x02, 0x03]));
     }
 
-    #[test]
-    fn test_path_data_conversions() {
-        let path_data = PathData {
-            flow_vertices: vec![Address::ZERO],
-            flow_edges: vec![(1, U192::from(1000u64))],
-            streams: vec![(0, vec![0], vec![0x01, 0x02])],
-            packed_coordinates: vec![0x03, 0x04],
-            source_coordinate: 0,
-        };
+    // #[test]
+    // fn test_path_data_conversions() {
+    //     let path_data = PathData {
+    //         flow_vertices: vec![Address::ZERO],
+    //         flow_edges: vec![(1, U192::from(1000u64))],
+    //         streams: vec![(0, vec![0], vec![0x01, 0x02])],
+    //         packed_coordinates: vec![0x03, 0x04],
+    //         source_coordinate: 0,
+    //     };
 
-        // Test individual conversions
-        let edges = path_data.to_flow_edges();
-        assert_eq!(edges.len(), 1);
-        assert_eq!(edges[0].streamSinkId, 1);
-        assert_eq!(edges[0].amount, U192::from(1000u64));
+    //     // Test individual conversions
+    //     let edges = path_data.to_flow_edges();
+    //     assert_eq!(edges.len(), 1);
+    //     assert_eq!(edges[0].streamSinkId, 1);
+    //     assert_eq!(edges[0].amount, U192::from(1000u64));
 
-        let streams = path_data.to_streams();
-        assert_eq!(streams.len(), 1);
-        assert_eq!(streams[0].sourceCoordinate, 0);
-        assert_eq!(streams[0].flowEdgeIds, vec![0]);
-        assert_eq!(streams[0].data, Bytes::from(vec![0x01, 0x02]));
+    //     let streams = path_data.to_streams();
+    //     assert_eq!(streams.len(), 1);
+    //     assert_eq!(streams[0].sourceCoordinate, 0);
+    //     assert_eq!(streams[0].flowEdgeIds, vec![0]);
+    //     assert_eq!(streams[0].data, Bytes::from(vec![0x01, 0x02]));
 
-        // Test combined conversion
-        let (vertices, edges, streams, coords) = path_data.to_contract_params();
-        assert_eq!(vertices, vec![Address::ZERO]);
-        assert_eq!(edges.len(), 1);
-        assert_eq!(streams.len(), 1);
-        assert_eq!(coords, Bytes::from(vec![0x03, 0x04]));
-    }
+    //     // Test combined conversion
+    //     let (vertices, edges, streams, coords) = path_data.to_contract_params();
+    //     assert_eq!(vertices, vec![Address::ZERO]);
+    //     assert_eq!(edges.len(), 1);
+    //     assert_eq!(streams.len(), 1);
+    //     assert_eq!(coords, Bytes::from(vec![0x03, 0x04]));
+    // }
```

Comment:
```suggestion
```

Reply:
Created an issue to improve tests: https://github.com/deluXtreme/circles-rs/issues/6